common/JailUtil.cpp

```diff
@@ -574,7 +574,7 @@ bool updateDynamicFilesImpl(const std::string& sysTemplate)
         FileUtil::Stat dstStat(dstFilename);
 
         // Is it outdated?
-        if (dstStat.isUpToDate(srcStat))
+        if (FileUtil::Stat::isUpToDate(srcStat, srcFilename, dstStat, dstFilename))
```

Comment:
This looks cumbersome. I suppose JailUtil being the only place we use `isUpToDate()` it's acceptable. Although I'm not sure removing `_path` is such a big win. Perhaps I forgot the reason behind it.